config_defaults_inc.php

```diff
@@ -541,6 +541,20 @@
 $g_validate_email = ON;
 
 /**
+ * Enable support for logging in by email and password, in addition to
+ * username and password.  This will only work as long as there is a single
+ * user with the specified email address and the email address is not blank.
+ *
+ * @global integer $g_email_login_enabled
+ */
+$g_email_login_enabled = ON;
+
```

Comment:
Would like to see default OFF as the new functionality introduces
- less security
- decreased performance, stresses server/database especially in SOAP scenario